public/app/plugins/datasource/cloudwatch/components/QueryEditor.tsx

```diff
@@ -112,30 +112,30 @@ export class QueryEditor extends PureComponent<Props, State> {
       <>
         <QueryInlineField label="Region">
           <Segment
-            value={query.region || 'Select region'}
+            value={this.toOption(query.region || 'Select region')}
```

Comment:
this makes it a bit harder to use the Segment as you have to do this everywhere, maybe supporting `T | SelectValue<T>`  will make it easier

Reply:
I agree! Will fix